src/Common/Simplify.elm

```diff
@@ -0,0 +1,62 @@
+module Common.Simplify exposing (ruleConfig)
```

Comment:
Similar code, but not identical, they don't read the same fields and don't produce messages in the same way. I would leave them separate, at least for now. I also do need to add unit tests for these, I guess I can do it once we commit to use them.

> A list of rules, along with their associated decoders and message formatters would also work I think

That's not a bad idea, but the exercise rules don't need decoders since they all use the same `decodeComment`.
Maybe `type AnalyzerRule = CustomRule Rule | ImportedRule Rule (Decoder Comment)`?